src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -333,8 +333,7 @@ public static OpenLoadFlowParameters load(PlatformConfig platformConfig) {
         return parameters;
     }
 
-    public static OpenLoadFlowParameters load(Map<String, String> properties) {
-        OpenLoadFlowParameters parameters = new OpenLoadFlowParameters();
+    public static OpenLoadFlowParameters load(OpenLoadFlowParameters parameters, Map<String, String> properties) {
```

Comment:
I think we can keep the old one and add this one as a "not static" method (then the old one can call the new one)